homework01/test/examples/Main.hs

```diff
@@ -1,15 +1,56 @@
-module Main
-  ( main
-  )
-  where
-
--- doctest
-import qualified Test.DocTest as DocTest
+module Main 
+  ( main )
+where
 
+import Basis
+import Test.Hspec
 
 main :: IO ()
-main =
-  DocTest.doctest
-    [ "-isrc"
-    , "src/Basis.hs"
-    ]
+main = hspec $ do
+
+  describe "toDigits" $ do
+    it "converts positive integers to a list of digits" $
```

Comment:
@jmorocho Some of these describe a context. Could you please verify and add `context`? http://hspec.github.io/writing-specs.html

```haskell
describe "toDigits" $ do
    context "when digits are positive integers" $ do
      it "converts to a list of digits" $ do
```